packages/fxa-auth-db-mysql/lib/db/mem.js

```diff
@@ -1497,6 +1498,55 @@ module.exports = function (log, error) {
       })
   }
 
+  Memory.prototype.createAccountSubscription = function (uid, subscriptionId, productName, createdAt) {
+    uid = uid.toString('hex')
+    return getAccountByUid(uid).then(() => {
```

Comment:
You should be able to use async functions to collapse the promises

Reply:
I can do that here, though FWIW the rest of this module seems to follow this pattern. I didn't want to do anything too different in case there was a reason not to.

Reply:
Reworked to use async/await